src/main/java/org/mule/extension/sftp/api/random/alg/PRNGAlgorithm.java

```diff
@@ -17,7 +17,7 @@
 
 public enum PRNGAlgorithm {
 
-  AUTOSELECT("AUTOSELECT", new SingletonRandomFactory(SecurityUtils.getRandomFactory())), SHA1PRNG("SHA1PRNG",
```

Comment:
[query] Is there any other any mulesoft way to mark things deprecated, possible which can strike the word horizontal or something?

Reply:
I don't think so